getting-started/choosing-a-development-host/mac-osx.md

```diff
@@ -1,4 +1,69 @@
 # Mac OSX
 
-Under development
+You can run Rocket.Chat for development on  Mac OSX. The following instructions has been tested on  M1 Macs.
 
+1. Install Meteor 
+
+    Currently Meteor don’t support Apple M1 native binaries as the latest meteor release uses Node.js 14 which doesn’t have support for it until now.
+
+    You can use Rosetta and enter the below command.
+
+    `arch -x86_64 npm install -g meteor`
+
+    or select Terminal in the Applications folder, press CMD(⌘)+I and check the “Open using Rosetta” option then using this command.
+
+    `npm install -g meteor`
+
+    (If your user doesn’t have permission to install global binaries, and you need to use sudo, it’s necessary to append –unsafe-perm to the below command:
+     `sudo npm install -g meteor --unsafe-perm` )
+     
+      There is no need to install `node` or `npm`, as meteor already includes them. Verify by:
+      
+      `meteor node -v`
+      
+      `meteor npm -v`
+
+   
+2) Fork the repository ` https://github.com/RocketChat/Rocket.Chat.git` and then clone the forked repository in your system.
+
+3) Install Modules
+
+    `cd Rocket.Chat`
+
+    `meteor npm install`
+
+4) Start building \(the first build can _take 10 or more minutes_, and you may see various warnings or minor errors -- please be patient; subsequent dev builds after the first will be 5 minutes or less\)
+
+   `meteor npm start`
+
+When the server is ready, you will see a box with "Server Running" title:
+
+
+This means that a Rocket.Chat server is running from your computer. To access the server, navigate to
+
+`http://localhost:3000`
+
+If you face any error like `npm ERR! Failed at the sharp@0.22.1 install script.` while installing modules.
+
+```
+1) Delete node_modules folder and package-lock.json
+2) Install sharp first (If it still throws an error lower you node version to v12 and
+   install sharp then back to same node version)
+3) meteor npm install
+4) meteror npm start
+  (It might prompt you to do npm rebuild if you used lower version of node to install sharp. Do the rebuild
+  and then meteor npm start)
+```
```

Comment:
I don't like it. And wouldn't want to add to our documentation. Fixing my other comments will easily resolve what **you experienced** here :)